src/main/kotlin/bandcampcollectiondownloader/Args.kt

```diff
@@ -41,6 +41,15 @@ data class Args(
         var jobs: Int = 4,
 
         @CommandLine.Option(names = ["-v", "--version"], versionHelp = true, description = ["Display the version and exits."])
-        var version: Boolean = false
+        var version: Boolean = false,
+        
+        @CommandLine.Option(names = ["-ft", "--filter-title"], usageHelp = false, description = ["only downloads releases matching the specified string"])
```

Comment:
Can you add a capital letter (s/only/Only) and a period, to match other descriptions?
Also to be better aligned with the name of your option, I would replace `the specified string` by `the specified title`.

Reply:
done, see https://github.com/TypicalFence/bandcamp-collection-downloader/commit/eef5a403952a0abf7877b997d2a4ca942b780240